dts/bindings/test/vnd,temperature-sensor.yaml

```diff
@@ -23,3 +23,11 @@ properties:
 
     clocks:
       required: true
+
+    pwms:
```

Comment:
I feel like this is kind of a weird place to put these. What about vnd,phandle-holder? Just bikeshedding a bit.

Reply:
Changed.